source/AndroidXProject.cshtml

```diff
@@ -94,9 +94,9 @@
 
   <ItemGroup>
     <None Include="@(Model.NuGetPackageId).targets" Pack="True" PackagePath="build\monoandroid90" />
-    <None Include="@(Model.NuGetPackageId).targets" Pack="True" PackagePath="build\net6.0-android30.0" />
+    <None Include="@(Model.NuGetPackageId).targets" Pack="True" PackagePath="build\net6.0-android$(TargetPlatformVersion)" />
```

Comment:
Unfortunately that didn't work, `$(TargetPlatformVersion)` must be blank at the time when this runs:
```
error NU1012: Some included files are included under TFMs which are missing a platform version: net6.0-android
```